addon/components/polaris-pagination.js

```diff
@@ -118,56 +111,58 @@ export default Component.extend({
    * @default no-op
    * @public
    */
-  onPrevious() {},
+  onPrevious() {}
 
   /** @private */
-  handleMouseUpByBlurring,
+  handleMouseUpByBlurring = handleMouseUpByBlurring;
 
   /** @private */
-  isPreviousDisabled: not('hasPrevious').readOnly(),
+  @(not('hasPrevious').readOnly())
+  isPreviousDisabled;
 
   /** @private */
-  isNextDisabled: not('hasNext').readOnly(),
+  @(not('hasNext').readOnly())
+  isNextDisabled;
 
   /** @private */
-  hasNextKeysConfigured: computed('nextKeys.[]', function() {
-    if (isEmpty(this.nextKeys)) {
+  @(computed('nextKeys.[]').readOnly())
+  get hasNextKeysConfigured() {
+    let { nextKeys } = this;
```

Comment:
no need to change this, but I'm not sure, generally speaking, it's worth to have this declaration when it's used in 1-2 places and especially if the attr name is short (doesn't makes code harder to read)